docs/b2b-edition/specs/storefront/storefront/rfq.yaml

```
@@ -1,2118 +1,1906 @@
-                      quoteId:
-                        type: number
-                      quoteUrl:
-                        type: string
-                        minLength: 1
-                    required:
-                      - quoteId
-                      - quoteUrl
-                required:
-                  - code
-                  - message
-                  - data
-                x-examples:
-                  example-1:
-                    code: 200
-                    message: SUCCESS
-                    data:
-                      quoteId: 195
-                      quoteUrl: 'https://sylvia-apptest-store.mybigcommerce.com/quote/?id=195&date=1622618606'
-              examples:
-                example-1:
-                  value:
-                    code: 200
-                    message: SUCCESS
-                    data:
-                      quoteId: 195
-                      quoteUrl: url
+                allOf:
+                  - $ref: "#/components/schemas/responseSuccess"
+                  - properties:
+                      data:
+                        properties:
+                          list:
+                            type: array
+                            items:
+                              $ref: "#/components/schemas/quoteData_LIST"
+                          pagination:
+                            type: object
+                            properties:
+                              totalCount:
+                                type: integer
+                                example: "250"
+                                description: "The total number of items in the full response."
+                              offset:
+                                type: integer
+                                default: "0"
+                                description: "The number of eligible results skipped before the first result in the response."
+                              limit:
+                                type: integer
+                                default: "10"
+                                description: "The number of records returned per page."
+    post:
+      tags:
+        - Quotes
+      summary: "Create a Quote"
+      operationId: post-rfq
+      description: |-
+        Creates a sales quote from the storefront and submits it to the B2B Edition control panel.
+
+        Depending on your store's Quote settings configuration, this can be used anonymously to create a quote as a guest shopper, or authenticated with a B2C storefront customer or B2B Company user account. Quotes created with this endpoint share the same [minimum required fields](https://developer.bigcommerce.com/b2b-edition/apis/rest-management/quote/quotes#minimum-required-fields) as the Create a Quote endpoint in the Server to Server API.
+
+        This endpoint cannot be used to create a draft quote which can be edited further in the Buyer Portal before submitting it to a sales representative.
```

I think this could bear some further explanation, making it clear what we're referring to as a "draft quote" and how it's actually tracked in the Buyer Portal app prior to being submitted with this equivalent GQL mutation.

---

Just chatted with Terra more about this.  In this context, I think it's worth getting into:

- A quote in draft status in the Buyer Portal is not the same as a quote in Draft status on the backend
- This endpoint is only to be used once a full quote is ready to submit.
- Keeping track of a quote in progress on the frontend is the app's responsibility. (And we can briefly mention how the Buyer Portal does it.)

---

```suggestion
        ### Draft Quotes
        
        The Buyer Portal allows buyers to save "draft" quotes" that they can edit further until they are ready to submit the quote to their sales representative. Unlike draft quotes in the B2B Edition control panel, the built-Buyer Portal uses local storage, via the [Redux](https://redux.js.org/) library, to store a draft quote's details.
              
        The Create a Quote endpoint should only be used once a quote is ready to submit to a sales representative. Your application must save quote details in its own storage in order to support draft quotes.
```

@CNanninga Let me know what you think of the above language.

---

The only thing I would add is a note making it clear that this is not the same thing as a quote created within the control panel and currently in Draft status.